build.gradle

```diff
@@ -0,0 +1,135 @@
+buildscript {
+    ext {
+        springBootVersion = '2.0.5.RELEASE'
+    }
+    repositories {
+        mavenLocal()
+        jcenter()
+
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+    }
+}
+
+plugins {
+    id 'idea'
+    id 'application'
+    id 'com.github.ben-manes.versions' version '0.20.0'
+    id 'com.jfrog.bintray' version '1.8.0'
+    id 'info.solidsoft.pitest' version '1.3.0'
+    id 'io.spring.dependency-management' version '1.0.5.RELEASE'
+    id 'jacoco'
+    id 'org.sonarqube' version '2.5'
+    id 'org.springframework.boot' version '1.5.12.RELEASE'
+    id 'pmd'
+}
+
+group = 'uk.gov.hmcts.reform'
+version = '1.0.0'
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+mainClassName = 'uk.gov.hmcts.reform.fpl.ccddatamigration.CcdDataMigrationApplication'
+
+compileJava {
+    options.compilerArgs << '-parameters' << '-Xlint:deprecation'
+}
+
+compileTestJava {
+    options.compilerArgs << '-Xlint:deprecation'
+}
+
+repositories {
+    mavenLocal()
+    jcenter()
+
+    maven { url 'https://repo.spring.io/libs-milestone' }
+    maven { url "https://dl.bintray.com/hmcts/hmcts-maven" }
+    maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+distributions {
+    main {
+        contents {
+            from(file("$projectDir/lib/AI-Agent.xml")) {
+                into "lib"
+            }
+        }
+    }
+}
+
+def versions = [
+        ccdStoreClient: '4.4.1',
+        commonsLang3: '3.7',
+        commonsIo: '2.5',
+        feignHttpClient: '9.5.1',
+        gradlePitest: '1.3.0',
+        jsonAssert: '1.2.3',
+        junit: '4.12',
+        lombok: '1.16.16',
+        pitest: '1.3.2',
```

Comment:
Lots of needed reps, like mutation testing framework for example. Other dips would be good to update as well.

Reply:
So much rubbish, I guess due to effort required we can clean that later.